ui/app/components/global-search/control.js

```diff
@@ -57,32 +58,40 @@ export default class GlobalSearchControl extends Component {
     const allTaskGroupResults = results.Matches.groups || [];
     const allCSIPluginResults = results.Matches.plugins || [];
 
-    const jobResults = allJobResults.slice(0, MAXIMUM_RESULTS).map(({ ID: name, Scope: [ namespace, id ]}) => ({
-      type: 'job',
-      id,
-      namespace,
-      label: name,
-    }));
-
-    const nodeResults = allNodeResults.slice(0, MAXIMUM_RESULTS).map(({ ID: name, Scope: [ id ]}) => ({
-      type: 'node',
-      id,
-      label: name,
-    }));
-
-    const allocationResults = allAllocationResults.slice(0, MAXIMUM_RESULTS).map(({ ID: name, Scope: [ , id ]}) => ({
-      type: 'allocation',
-      id,
-      label: name,
-    }));
-
-    const taskGroupResults = allTaskGroupResults.slice(0, MAXIMUM_RESULTS).map(({ ID: id, Scope: [ namespace, jobId ]}) => ({
-      type: 'task-group',
-      id,
-      namespace,
-      jobId,
-      label: id,
-    }));
+    const jobResults = allJobResults
+      .slice(0, MAXIMUM_RESULTS)
+      .map(({ ID: name, Scope: [namespace, id] }) => ({
+        type: 'job',
+        id,
+        namespace,
+        label: `${name} > ${namespace}`,
```

Comment:
The hierarchy seems reversed here. I think I would expect to see `${namespace} > ${job}` since a job belongs to a namespace.